lib/mirage/mirage.mli

```diff
@@ -764,6 +764,104 @@ val default_argv : argv impl
 val no_argv : argv impl
 (** [no_argv] Disable command line parsing and set argv to [|""|]. *)
 
+(** {2 Mimic device} *)
+
+(** A {i mimic} device is a {i context} which contains few informations to let
```

Comment:
rewording suggestion: "A {i mimic} device is a {i context} allowing the user to [connect]/{!Mimic.resolve}/allocate a new resource to communicate with a peer" . If this is too drastic a change, "few informations" is strange and I'd at least change that to "information".